.gitignore

```diff
@@ -1,3 +1,5 @@
+*.json
+*.arrow
 *.pkl
 ckpts*
 .deepspeed_env
```

Comment:
Add more specific commit messages for better understanding of the changes made in each PR.